src/App.jsx

```diff
@@ -162,10 +160,9 @@ function App() {
 
             {/* ----- Canvas ------ */}
             <canvas
-              id="draw"
-              className={`whiteboard bg-slate-950 w-screen mt-[4vh] rounded-[0.6rem] shadow-md shadow-black dark:shadow-black dark:shadow-lg ${
-                isDrawing ? "cursor-crosshair" : "cursor-default pointer-events-none"
-              }
+              // id="draw"
+              className={`whiteboard bg-slate-950 rounded-[0.6rem] shadow-md shadow-black dark:shadow-black dark:shadow-lg ${isDrawing ? "cursor-crosshair" : "cursor-default pointer-events-none"
+                }
```

Comment:
I would recommend you to remove `isDrawing` from `className`, because the `isDrawing` is always gonna be `true`. We don't need conditional rendering here.

So you can simply use `cursor-crosshair` for canvas `className`.

```suggestion
className="whiteboard bg-slate-950 rounded-[0.6rem] shadow-md shadow-black dark:shadow-black dark:shadow-lg cursor-crosshair"
```